New serverless pattern - cdn-dash-sam (#1780)
aws-samples/serverless-patterns: cdn-dash-sam/README.md

# Create Cloudwatch custom metrics for API Gateway custom domain names using SAM

> This pattern deploys Amazon Eventbridge rule and a AWS Lambda function that creates custom metrics in Amazon CloudWatch for the API mappings of the custom domain names that are present in the region.

Reviewer: ```suggestion
This pattern deploys an Amazon EventBridge rule and an AWS Lambda function that creates custom metrics in Amazon CloudWatch for the API mappings of custom domain names that in a region.
```

> Learn more about this pattern at Serverless Land Patterns: << Add the live URL here >>

Reviewer: ```suggestion
Learn more about this pattern at Serverless Land Patterns: https://serverlessland/patterns/apigw-custom-metrics
```
